kouncil-frontend/src/app/demo/demo.component.html

```diff
@@ -0,0 +1,3 @@
+<div>
+  This is a demo version of Kouncil. Get full version at&nbsp;<a href="https://kouncil.io" target="_blank">kouncil.io</a> <mat-icon aria-hidden="false">open_in_new</mat-icon>
```

Comment:
jestem otwarty na wszelkie sugestie co do treści ;)

Reply:
dałbym full -> the full

Reply:
done ;)